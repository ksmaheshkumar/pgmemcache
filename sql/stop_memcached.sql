\! kill `cat $HOME/.pgmemcache-memcached.pid`
