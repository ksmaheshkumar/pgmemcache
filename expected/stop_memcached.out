\! service memcached stop
Stopping memcached: memcached.
